h/static/styles/annotation.scss

```diff
@@ -7,29 +7,57 @@
 $annotation-card-left-padding: 10px;
 
 .annotation-card {
-  background-color: white;
+  box-shadow: 0px 1px 1px 0px rgba(0, 0, 0, 0.10);
   border-radius: 2px;
-  border: 1px solid $color-mercury;
-  padding-left: 12px;
-  padding-right: 12px;
-  padding-top: 12px;
-  padding-bottom: 15px;
+  padding: $layout-h-margin;
+  background: $white;
+}
+
+.annotation-card:hover {
+  box-shadow: 0px 2px 3px 0px rgba(0, 0, 0, 0.15);
+  border-radius: 2px;
```

Comment:
The border radius does not change on hover so we don't need to specify it again here.